web/src/i18n/pl/index.ts

```diff
@@ -1267,6 +1267,15 @@ Uwaga, konfiguracje tutaj podane, nie posiadają twojego klucza prywatnego. Musi
           label: 'Dozwolone grupy',
           placeholder: 'Wszystkie grupy',
         },
+        mfa_enabled: {
+          label: 'Włącz MFA',
+        },
+        keepalive_interval: {
+          label: 'Keepalive interval',
```

Comment:
Zgapiłem z WireGuard na macOS

```suggestion
          label: 'Utrzymanie połączenia',
```